app/src/Clone/CCServer.ts

```diff
@@ -38,38 +38,70 @@ export default class CCServer {
   config: Config;
   isActive = false;
   lastRequestTime = process.hrtime();
+  pendingProblems: ProblemData[] = [];
+  totalProblemsCount = 0;
+  receivedProblemsCount = 0;
+
   constructor(config: Config) {
     this.config = config;
     this.app.use(express.json());
     this.app.post("/", (request, response) => {
+	  const problemData: ProblemData = request.body;
+    const batchSize = problemData.batch?.size || 1;
+
+    if (this.totalProblemsCount === 0) {
+      	this.totalProblemsCount = batchSize;
+	  }
```

Comment:
there are some code formatting issue here (use spaces instead of tab)

Reply:
I'm sorry. I will re-format it asap. I think this issue comes from the fact that Vim used spaces.